content/terms-of-service/termsOfService.md

```diff
@@ -1,3 +1,494 @@
+				<br>
+				<li><p>&nbsp;&nbsp;You will only be eligible to receive Credits if you have paid all Fees on time, are not past due on any payments, and notify us of a request for Credits within 30 days of the end of the calendar month in which we have not met our SLA. We will issue Credits solely as an offset against future Fees that are payable by you to Company. You will not be entitled to a refund or other payment from Company in connection with any Credits you accrue. In no event will the total amount of Credits exceed 10% of the Fees payable by you for the corresponding month.</p></li>
+				<br>
+				<li><p>&nbsp;&nbsp;<b>Scheduled Downtime.</b> Scheduled Downtime will generally occur during the Maintenance Windows. Company will endeavor to provide notice at least eight hours in advance of any scheduled downtime occurring outside of the Maintenance Windows.</p></li>
+				<br>
+				<li><p>&nbsp;&nbsp;<b>Maintenance Windows.</b> Company has an optional weekly maintenance window on Sundays from 2:00 a.m. EST/EDT to 6:00 a.m. EST/EDT during which scheduled maintenance, upgrades and repairs can occur.</p>
+					<ol type="1">
+						<li><p>&nbsp;&nbsp;Usage of the maintenance window is scheduled according to the Company release calendar.</p></li>
+
+						<li><p>&nbsp;&nbsp;Company may also perform emergency maintenance in a non-standard maintenance window.</p>
+							<ol type="a">
+								<li><p>&nbsp;&nbsp;Company will use commercially reasonable efforts to perform emergency maintenance at the time of lowest use levels, as determined by web use logs from the previous month.</p></li>
+
+								<li><p>&nbsp;&nbsp;Emergency maintenance windows will last no longer than four (4) hours.</p></li>
+
+								<li><p>&nbsp;&nbsp;Company reserves the right to use two (2) emergency (non-scheduled) maintenance windows per year.  Emergency maintenance beyond these two (2) additional windows will be considered downtime.</p></li>
+
+								<li><p>&nbsp;&nbsp;Company will inform User about all relevant changes planned for the upcoming maintenance window no less than one (1) weeks prior to the maintenance window.</p></li>
+							</ol>
+						</li>
+					</ol>
+				</li>
+			</ol>
+		</li>
+	</ol>
+</p>
+
+<p><b>10.&nbsp;&nbsp;Feedback</b>To the extent that User or its End Users provide Company with any suggestions, ideas, enhancement requests, recommendations, or other feedback or information (collectively, “<b>Feedback</b>”), User hereby grants to Company a fully paid-up, royalty free, worldwide, perpetual, irrevocable, transferable, sub-licensable license to freely use, disclose, and otherwise exploit such Feedback, including but not limited to incorporation of the Feedback into future versions of the Storage Services without any compensation or other obligation to User. </p>
```

Comment:
Add space or new line after the title "10. Feedback"

Reply:
<img width="810" alt="Screen Shot 2020-03-27 at 4 34 38 PM" src="https://user-images.githubusercontent.com/10093024/77798330-09596b00-7049-11ea-8abe-bf5bc81c9cec.png">